rafthttp/transport.go

```diff
@@ -307,3 +329,23 @@ func (t *Transport) Resume() {
 		p.(Pausable).Resume()
 	}
 }
+
+// snapshotStore is the temporary store of snapshot. Caller could put one
+// snapshot into the store, and get it later.
+// snapshotStore stores at most one snapshot at a time.
```

Comment:
it stores the new one then. I should let it panic here to let the comment always be true.